virt/virt-2-5-release-notes.adoc

```diff
@@ -64,6 +64,8 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 //Placeholder for new content.
 
 //CNV-7170 - CDI import of container disks is faster and more efficient
+You can now use the Containerized Data Importer (CDI) to faster import `containerDisk` storage volumes from the container image registry and allocate storage capacity more efficiently. The CDI can pull a `containerDisk` image from the registry in about the same amount of time that it would
```

Comment:
Can we reword the first sentence to something like:
"Containerized Data Importer (CDI) has optimized importing `containerDisk` volumes from a container image registry.  Import speed is improved and storage capacity is used more efficiently.

It sounds awkward to me when I read "the CDI".  You can use the acronym CDI on it's own (without 'the') in this case.

Reply:
Tweaked the first sentence